acceptance_tests/features/steps/change_response_status.py

```diff
@@ -23,7 +23,7 @@ def create_collection_exercise(_, survey, period):
     now = datetime.utcnow()
     dates = {
         "mps": now + timedelta(seconds=5),
-        "go_live": now + timedelta(minutes=1),
+        "go_live": now + timedelta(minutes=2),
```

Comment:
@NickGrantONS Are we sure 2 minutes is enough, or is this a sort of trial and error process?

Reply:
trial and error, but every minute we add delays the whole acceptance test suite by 1 minute